### ISSUE TITLE
Airlock UI: New Request

### ISSUE THREAD

**Issue by jjgriff93** [microsoft/AzureTRE#2354]
Implement as per https://www.figma.com/file/Ncl8TJsFf2WKf3frrqTbtq/AzureTRE-OSS-UI?node-id=305%3A21271

Showing file information depends on #2267, which is currently blocked/low priority, so this can be omitted for now
